lib/action_subscriber/middleware/env.rb

```diff
@@ -30,7 +30,7 @@ def initialize(subscriber, encoded_payload, properties)
         @encoded_payload = encoded_payload
         @exchange = properties.fetch(:exchange)
         @headers = properties.fetch(:headers) || {}
-        @message_id = properties.fetch(:message_id)
+        @message_id = properties.fetch(:message_id) || ::Random.new.bytes(3).unpack("H*")[0]
```

Comment:
PS: I used `Random.new.bytes` instead of `SecureRandom.hex` because I'm assuming the overhead is less, but maybe it doesn't matter. Anyone know if there are concerns around running out of entropy or other concerns with `SecureRandom` vs `Random`?